lib/utils.js

```diff
@@ -1,52 +1,122 @@
+const Cache = require('./cache')
+const Mustache = require('mustache')
+const Needle = require('needle')
+const Winston = require('winston')
+const CronJob = require('cron').CronJob
+const Routes = require('./routes')
+const URL = require('url')
+const removeRoute = require('express-remove-route')
 
-var template = '{"title":"{{title}}","headers":{{{headers}}},"rows":[{{{row}}}],"width":{{width}},"height":1}';
-Mustache.parse(template);
+const template = '{"title":"{{title}}","headers":{{{headers}}},"rows":[{{{row}}}],"width":{{width}},"height":1}'
+Mustache.parse(template)
 
 // Fetch all data and add it to the cache
-exports.populateCache = function populateCache(url) {
-  Winston.info('Populating cache');
+let populateCache = exports.populateCache = function populateCache (url) {
+  Winston.info('Populating cache')
 
   // Fetch the facility data
-  Needle.get(url, function(err, res) {
+  Needle.get(url, function (err, res) {
     if (err) {
-      return Winston.error('Fetching facility data failed', {err: err.message});
+      return Winston.error('Fetching facility data failed', {err: err.message})
     }
 
     if (res.statusCode !== 200) {
-      return Winston.error('Non-200 status code: %d', res.statusCode);
+      return Winston.error('Non-200 status code: %d', res.statusCode)
     }
 
-    var title = res.body.title;
-    var headers = res.body.headers;
+    var title = res.body.title
+    var headers = res.body.headers
 
-    var batch = Cache.batch();
+    var batch = Cache.batch()
 
     // Add each facility to the cache
-    res.body.rows.forEach(function(row) {
-      var key = row[0];
+    res.body.rows.forEach(function (row) {
+      var key = row[0]
       if (!key) {
-        return Winston.error('Facility missing value', {facility: row});
+        return Winston.error('Facility missing value', {facility: row})
       }
       var body = Mustache.render(template, {
         title: title,
         headers: JSON.stringify(headers),
         row: JSON.stringify(row),
         width: headers.length
-      });
-      batch.put(key, body);
-    });
+      })
+      batch.put(key, body)
+    })
 
-    batch.write(function(err) {
+    batch.write(function (err) {
       if (err) {
-        return Winston.error('Storing facility data failed', {err: err.message});
+        return Winston.error('Storing facility data failed', {err: err.message})
       }
-      Winston.info('Finished populating cache');
-    });
-  });
-};
+      Winston.info('Finished populating cache')
+    })
+  })
+}
+
+let cronJobs = {}
+
+let addCronJob = exports.addCronJob = (pattern, timezone, url) => {
+  cronJobs[url] = new CronJob(pattern, populateCache.bind(null, url), true, timezone)
+}
+
+let removeCronJob = (url) => {
+  if (cronJobs[url]) {
+    cronJobs[url].stop()
+    delete cronJobs[url]
+  }
+}
+
+let addRoute = exports.addRoute = (app, path) => {
+  app.get(path, Routes.facilityCodeLookup)
+}
+
+let removeCacheForRoute = (app, path) => {
```

Comment:
Is there any value in having this function which just calls another one with the exact same arguments?